README.md

```diff
@@ -13,4 +13,4 @@ You can find the pages in **/docs/help-center/**.
 If you edit a page, make sure you edit the German localization in **/i18n/de/docusaurus-plugin-content-docs/current/help-center/**, too. 
 
 To add a page, you can copy and adjust an existing page and add a translation to **/i18n/de/docusaurus-plugin-content-docs/current/help-center/**. 
```

Comment:
I am not sure if this works. Normally we do the translation using crowdin. But lets try it out and see what happens.

Reply:
I can't use crowdin, I get the error that an api_token is missing